src/metrics.rs

```diff
@@ -4,6 +4,7 @@ use iroh_metrics::{Counter, MetricsGroup};
 
 /// Enum of metrics for the module
 #[allow(missing_docs)]
+#[allow(dead_code)]
```

Comment:
why is this dead code?

Reply:
This thing was a separate crate. I inlined it so I don't have to publish and maintain yet another crate. The dead code part is the friendly API of the crate, if you don't want to embed the entity manager into your own actor.

But I want to embed the entity manager, since I don't want client -> iroh-blobs main actor -> entity manager -> entity. Save one hop.

Reply:
I need to rethink what metrics to collect, many of those aren't even relevant anymore.